cdc/model/sink.go

```diff
@@ -25,6 +25,8 @@ import (
 	"go.uber.org/zap"
 )
 
+//go:generate msgp
```

Comment:
For generated data, can it be decoded by other programming languages?

Reply:
should be https://github.com/msgpack/msgpack/blob/master/spec.md